Client/metrics.yaml

```diff
@@ -574,7 +574,7 @@ legacy.ids:
     description: |
       The client id from legacy telemetry.
     send_in_pings:
-      - deletion_request
+      - deletion-request
```

Comment:
I will undo this change before merging. Just trying that build works fine on BB to run the tests

Reply:
This can not land on main (Xcode 11.6 or 11.7) or it will be broken. Once main uses Xcode 12, this change will be needed to have build working.